contracts/Configuration.sol

```diff
@@ -9,7 +9,7 @@ struct MarketplaceConfig {
 }
 
 struct CollateralConfig {
-  uint256 initialAmount; // amount of collateral necessary to fill a slot
+  uint256 minimalInitialAmount; // amount of collateral necessary to fill a slot
```

Comment:
I don't think that we need this anymore; a client is free to choose whatever amount of collateral they wish (even zero)

Reply:
Yeah, I was considerating removing it, but then realized that when the repair comes into game, then there should be always some collateral requested so the funds could be used for funding the repair?